gnat2goto/driver/tree_walk.adb

```diff
@@ -4244,81 +4269,247 @@ package body Tree_Walk is
       end;
    end Maybe_Make_Typecast;
 
+   --------------------------------
+   --  Warn_Unhandled_Construct  --
+   --------------------------------
+
+   procedure Warn_Unhandled_Construct (C : Construct; Mess : String) is
```

Comment:
When we don't handle something it'd help if we also printed the node we can't handle for debugging purposes

Reply:
done.